Components/avatar/avatar.css

```diff
@@ -0,0 +1,54 @@
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.avatar-container {
+  display: flex;
+  align-items: flex-end;
+  justify-content: space-around;
+  width: 40rem;
+}
+
+.avatar-xl {
+  border-radius: 50%;
+  width: 10rem;
+  height: 10rem;
+}
```

Comment:
I think instead of adding `border-radius` to all `avatar` classes, don't you think we should keep the default look of the avatars as square and create a class like `rounded` and use that in the HTML. 
```
.rounded {
  border-radius: 50%;
}
```
I understand that you have done the reverse.

Reply:
Okay I think this way will also  work , i will create common avatar class which will have the border-radius property, xl sm etc for different sizes , what do you think ?

Reply:
Okay. If you want to keep the default look of avatars as rounded, yes, you can do that.

Reply:
Yaah actually wanted the deafult to be rounded , can you approve changes now